master/pkg/schemas/expconf/hyperparameters_config_test.go

```diff
@@ -0,0 +1,30 @@
+package expconf
+
+import (
+	"encoding/json"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+func TestNestedHPs(t *testing.T) {
```

Comment:
it's not clear to me that this test tests anything not already tested in `schema_test.go`, where we also do round-trip-through-json testing.  If this test is in fact redundant, we should remove it.